local/protocol/local.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+// TODO(dbentley): integrate this into our go build
```

Comment:
I was sure it worked with Unix Domain Sockets, and Thrift wasn't clear on if it could.

Also, the previous bugs in the thrift implementation for Go made me feel uneasy.

I'm not convinced it's a perfect choice, which is why it's hidden and doesn't escape.